cypress/integration/form.spec.ts

```diff
@@ -0,0 +1,70 @@
+// const book_title = 'The Little Schemer';
+
+describe('Form Actions', () => {
+  beforeEach(() => {
+    cy.visit('/#/examples/form');
+  });
+
+  it('should be able to input all values', () => {
+    cy.get('mat-slide-toggle[formcontrolname="autosave"]').click();
```

Comment:
I'm not really a big fan of using `formcontrolname` in tests, because it shouldn't really matter to the user how we're naming our form fields or which "technology" we use.

I do prefer using visual aspects of the form that an end-user can see, for example labels and placeholders. For an example you can take a look at the [existing CRUD tests](https://github.com/tomastrajan/angular-ngrx-material-starter/blob/master/cypress/integration/crud.spec.ts) or in https://github.com/tomastrajan/angular-ngrx-material-starter/pull/420.

Reply:
I would rather change the tests when the text changes, instead of using `id`'s.
Because here again, the user sees the texts and interacts with the application by using the text, not the id's. If there is no text "available", I would use a property `data-testid` instead of `id`.

This is based on Kent C. Dodds's articles, e.g.: https://blog.kentcdodds.com/making-your-ui-tests-resilient-to-change-d37a6ee37269

Reply:
I made the selectors more generic. Other than the mat-error. ANy other feedback? thx

Reply:
I like it, thanks!